avro-schema/ingestion-header.avsc

```diff
@@ -52,6 +52,11 @@
             "type": "long",
             "logicalType": "timestamp-millis",
             "doc": "time range information for the shares in this batch."
+        },
+        {
+            "name": "packet_file_digest",
+            "type": "bytes",
+            "doc": "SHA-256 digest of the .avro file containing packets in this batch."
```

Comment:
As pointed out in another comment, should there be a field to identify the key used to sign the header?

Reply:
> How do we want to specify which public key was used when generating the signature, as it may happen that during a rotation the ingestion service issues some batches using the old key if is has not been updated yet?

Not a bad idea. What I wrote up in https://docs.google.com/document/d/1MdfM3QT63ISU70l63bwzTrxr93Z7Tv7EDjLfammzo6Q/edit#heading=h.4jef7v1c76vl is that clients should attempt to use all the keys listed in the peer's keyfile, but including a key fingerprint of some sort is better. We haven't yet settled on a plan of record for key exchanges so I think what I'd like to do is incorporate this suggestion into the design doc and then implement the corresponding schema changes once we've all agreed on how this will work. WDYT @manu-raghunath and @tlepoint?

Reply:
Sounds good.